plugins/PagePerformance/Columns/Base.php

```diff
@@ -37,6 +37,12 @@ public function onNewAction(Request $request, Visitor $visitor, Action $action)
             throw new InvalidRequestParameterException(sprintf('Value for %1$s can\'t be negative.', $this->getRequestParam()));
         }
 
+        // ignore obviously incorrect values (nothing should take more than 1 hour to load)
+        // refs https://github.com/matomo-org/matomo/issues/17035
+        if ($time > 3600000) {
```

Comment:
@sgiehl since we don't know what people are tracking it be better to go with some max value like 16million since people might be tracking apps, or videos, or all kind of things using such parameters.

Reply:
ok. changed hat.